includes/Classifai/Providers/Provider.php

```diff
@@ -249,20 +249,40 @@ public function render_prompt_repeater_field( array $args ): void {
 			$args['label_for']
 		);
 
-		$value = ( empty( $value ) && isset( $args['default_value'] ) ) ? $args['default_value'] : $value;
+		$prompts = empty( $prompts ) && isset( $args['default_value'] ) ? $args['default_value'] : $prompts;
 
-		$prompt_count = count( $value );
+		$prompt_count = count( $prompts );
 		$field_index  = 0;
 		?>
 
-		<?php foreach ( $value as $prompt ) : ?>
-			<?php $is_default_prompt = 1 === $prompt['default']; ?>
+		<?php foreach ( $prompts as $prompt ) : ?>
+			<?php
+			$is_default_prompt  = ( isset( $prompt['default'] ) && 1 === $prompt['default'] ) || 1 === $prompt_count;
+			$is_original_prompt = isset( $prompt['original'] ) && 1 === $prompt['original'];
+			?>
 
 			<fieldset class="classifai-field-type-prompt-setting">
+				<?php if ( $is_original_prompt ) : ?>
+					<p class="classifai-original-prompt">
+						<?php
+						printf(
+							/* translators: %1$s is replaced with <strong>; %2$s with </strong>; %3$s with prompt. */
+							esc_html__( '%1$sClassifAI default prompt%2$s: %3$s', 'classifai' ),
+							'<strong>',
+							'</strong>',
+							esc_html( $placeholder )
+						);
+						?>
+					</p>
+				<?php endif; ?>
+
 				<input type="hidden"
 					name="<?php echo esc_attr( $field_name_prefix . "[$field_index][default]" ); ?>"
 					value="<?php echo esc_attr( $prompt['default'] ?? '' ); ?>"
 					class="js-setting-field__default">
+				<input type="hidden"
+					name="<?php echo esc_attr( $field_name_prefix . "[$field_index][original]" ); ?>"
+					value="<?php echo esc_attr( $prompt['original'] ?? '' ); ?>">
```

Comment:
@dkotter I suggest use the trinary short syntax `?:` instead of the null coalescing operator`??`, because `original` value can not be `null`, right?

Reply:
So short ternary's aren't allowed by our PHPCS config so we would need to change this to `! empty( $prompt['original'] ? $prompt['original'] ? ''` which I'm fine with changing that. The goal here was to be defensive for a situation where this array value wasn't set (which should never happen) and using the null coalescing operator would catch that.